server/webhook.go

```diff
@@ -246,6 +252,18 @@ func validateGitHubWebhookSignature256(signature, key string, body []byte) (bool
 	return subtle.ConstantTimeCompare([]byte(signature), []byte(got)) == 1, nil
 }
 
+// parseBranchNameFromGitHubRefs is for GitHub.
+// GitLab can configure branches that trigger push events, but GitHub cannot do it. So we need parse the branch name from refs field in request.
```

Comment:
> GitLab can configure branches that trigger push events, but GitHub cannot do it.
I'm not sure what this means.

Reply:
More detail in linear issue 1219.

Reply:
Thanks. I tried to rewrite the comment and make it more clear. PTAL